src/reducers/noteReducer.js

```diff
@@ -1,7 +1,11 @@
 const noteReducer = (state, { type, payload }) => {
   switch (type) {
-    case "GET_NOTES":
-      return [...payload];
+    case "SET_NOTES":
+      return { ...state, notes: payload };
+    case "SET_ARCHIVED":
+      return { ...state, archives: payload };
+    case "SET_NOTES_AND_ARCHIVE":
+      return { ...state, notes: payload.notes, archives: payload.archives };
```

Comment:
Again, Suggestions. your code is so clean. please make a const variable for it too.(actions)

Reply:
Yes, I'll do that. Thanks.